scripts/java-api-check.json

```diff
@@ -32,6 +32,7 @@
             "com\\.mapbox\\.maps\\.renderer\\.egl\\.EGLConfigChooser",
             "com\\.mapbox\\.maps\\.renderer\\.egl\\.EGLCore",
             "com\\.mapbox\\.maps\\.renderer\\.gl\\.PixelReader",
+            "com\\.mapbox\\.maps\\.renderer\\.widget\\.BitmapWidgetRenderer",
```

Comment:
interface `WidgetRenderer` could also be added here, it's internal

Reply:
Added